contracts/script/DeployLiquity2.s.sol

```diff
@@ -789,6 +771,46 @@ contract DeployLiquity2Script is DeployGovernance, UniPriceConverter, StdCheats,
             _deployZappers(contracts.addressesRegistry, contracts.collToken, _boldToken, _usdcCurvePool);
     }
 
+    function _deployPriceFeed(uint256 _branch, address _collTokenAddress, address _borroweOperationsAddress)
```

Comment:
Could even eliminate the `_branch` parameter, and identify the branch by `_collTokenAddress`.